dhcp/unikernel.ml

```diff
@@ -0,0 +1,85 @@
+open V1_LWT
+open Lwt
+
+(* IP Configuration, all you need besides dhcpd.conf. *)
+let ipaddr = Ipaddr.V4.of_string_exn "192.168.1.5"
```

Comment:
On Wednesday, 11 November 2015, Hannes Mehnert <notifications@github.com>
wrote:

> In dhcp/unikernel.ml
> <https://github.com/mirage/mirage-skeleton/pull/105#discussion_r44581922>:
>
> > @@ -0,0 +1,85 @@
> > +open V1_LWT
> > +open Lwt
> > +
> > +(* IP Configuration, all you need besides dhcpd.conf. *)
> > +let ipaddr = Ipaddr.V4.of_string_exn "192.168.1.5"
>
> shouldn't this be passed in from config.ml?
>
> —
>
>
>

Yep, I couldn't figure how, should I pass a module with just that to the
main in unikernel ?